api/xds_protocol.rst

```diff
@@ -0,0 +1,456 @@
+xDS REST and gRPC protocol
+==========================
+
+Envoy discovers its various dynamic resources via the filesystem or by
+querying one or more management servers. Collectively, these discovery
+services and their corresponding APIs are referred to as *xDS*.
+Resources are requested via *subscriptions*, by specifying a filesystem
+path to watch, initiating gRPC streams or polling a REST-JSON URL. The
+latter two methods involve sending requests with a :ref:`Discovery Request <envoy_api_msg_DiscoveryRequest>`
```

Comment:
Why did you add a space here between "Discovery" and "Request" (and "Response" below)? Ideally we keep the content exactly the same as the previous, no changes to things like naming etc.

Reply:
It was by mistake.  I removed the spaces now